particles/PipeApps/AndroidAutofill.arcs

```diff
@@ -0,0 +1,26 @@
+schema AutofillRequest
+  Number autofillId
+  Text hint
+
+schema AutofillResponse
+  Number autofillId
+  Text suggestion
+
+// An Android particle which powers the Android AutofillService.
+particle AutofillParticle in './source/AutofillParticle.java'
+  out [AutofillRequest] request
+  // TODO: Provide a slot for the response instead of a handle.
+  in [AutofillResponse] response
+
+// A dummy JS particle which handles the Autofill requests.
+particle DummyAutofillResponder in './source/DummyAutofillResponder.js'
+  in [AutofillRequest] request
+  out [AutofillResponse] response
+
+recipe AndroidAutofill
+  AutofillParticle
+    request -> request
```

Comment:
My knowledge might be outdated. So we don't need to create these handles explicitly now?

Reply:
The handles fates default to `create` when manifest is loaded: https://github.com/PolymerLabs/arcs/blob/master/src/runtime/manifest.ts#L966